### ISSUE TITLE
Chart inside a scrollable div: persistent crosshair offset when scrolling the div

### ISSUE THREAD

**Issue by olejorgenb** [highcharts/highcharts#13488]
#### Live demo with steps to reproduce

https://jsfiddle.net/95crzfu7/2/

Srcoll the inner container (overflow: auto) while the pointer is inside the chart, observe that the crosshair get an vertical offset. Only way to reset the offset is to move the cursor out and into the chart again.

Probably related: #12799 (I haven't tried yet, but I suspect the linked SO answer would work: https://stackoverflow.com/a/59769430/1517969)

#### Product version

Highcharts JS v8.1.0 (2020-05-05)

#### Affected browser(s)

Tested on recent versions of Firefox and Chrome

**Comment by pawelfus** [comment]
Thanks for reporting this bug @olejorgenb 

**Additional workaround:**
Demo: https://jsfiddle.net/BlackLabel/djh84vqo/
Plugin: 
```js
(function(H) {
  H.wrap(
    H.Pointer.prototype,
    'getChartPosition',
    function(p) {
      delete this.chartPosition;

      return p.apply(this, Array.prototype.slice.call(arguments));
    }
  )
})(Highcharts);
```

*Internal note:*
In theory, we do clear this cache on `this.scrollingContainer`'s `scroll` event. We need to check why it does not solve the issue.

**Comment by olejorgenb** [comment]
Yes, I ended up clearing all the `Highchart.charts[*].pointer.chartPosition` on the `scroll` event :) (assuming there's some performance hit in recalculation the position on every mouse move)

**Comment by khlieng** [comment]
Fixed in #14768.
